### ISSUE TITLE
Rails 5 version?

### ISSUE THREAD

**Issue by sandergroen** [phatworx/devise_security_extension#199]
Any idea when we can expect a new version for Rails 5?

**Comment by fab-nulogy** [comment]
there's a PR here https://github.com/phatworx/devise_security_extension/pull/188

can't say I've seen much activity at all from the maintainers since the r5 release though ._.

stopped waiting on them and am pulling my gem from that PR's latest commit directly

**Comment by sandergroen** [comment]
Maybe they can add some active maintainers 😄

**Comment by rquant** [comment]
Still no update on this? Is there an alternative way to validate password complexity without writing your own?

**Comment by jstumbaugh** [comment]
There is also PR #215 to try and upgrade to Rails 5. No movement on it in the last 2 months though.

**Comment by nynhex** [comment]
Someone should pick this up, Rails is at `5.2` now and this gem is broken in multiple places.  Does anyone want to collaborate with the maintainers?

**Comment by oniofchaos** [comment]
I would love to but I don't know any of them. There's some forks with fixes for things like Rails 5 and Ruby 2.4.

**Comment by arturoherrero** [comment]
OK, let's assume that the project is dead. They are not giving any information about the project status, possible maintainers, etc...

What is the new _official_ fork that we can select @oniofchaos? If the whole community moves to one fork, maybe at some point the owners of this gem link to the new gem in the README.

**Comment by oniofchaos** [comment]
There's the fork I've been maintaining for my company's needs. https://github.com/q-centrix/devise_security_extension

There's also https://github.com/natebird/devise_security_extension/ maintained by another user who has needed to make updates to the gem.

**Comment by danielricecodes** [comment]
Its frustrating that such a useful gem for a canonical Rails plugin is abandoned like this and isn't being updated for Rails 5.

**Comment by sandergroen** [comment]
Sounds like a good idea @oniofchaos. But which one should be the new official fork? Are we using your companies version?
I have a pending pull request for this repo and I would like to make a new one to add a new generator feature.

**Comment by natebird** [comment]
I'd prefer making a new group so the community can own the repository and we avoid this happening again. That way as maintainers come and go the repository can be passed to others without fussing about it being under such and such user's namespace.

I'd be glad to create a group for it and anyone who wants to maintain the gem I'll add them. In my view, for a repository such as this, if you make any significant pull request and want access to the repository then you can have it. We'll take all the help we can get. ;-)

**Comment by sandergroen** [comment]
Sounds like a plan!

**Comment by natebird** [comment]
I forked the gem and created a new repo. https://github.com/devise-security/devise-security. Let me know if you'd like to be a maintainer.

The tests pass but I haven't played with it in my app yet. I'm still in process of setting up Travis and some other bots to help with maintaining code quality. I'm not going to port the issues, so if you have one outstanding then add it to the new repo.

Let me know if you have any questions or concerns. I'm just winging it. :-)

**Comment by oniofchaos** [comment]
I would like to be added to the group, thanks!

**Comment by oniofchaos** [comment]
Side note, created https://github.com/plataformatec/devise/issues/4619 to ask the Devise folks what they think about getting this merged into Devise.

**Comment by danielricecodes** [comment]
@natebird - I just plugged it into my app and will let you know if I run into any issues.  

```ruby
gem 'devise-security', github: 'devise-security/devise-security'
```

**Comment by arturoherrero** [comment]
Thanks @natebird for the new _official_ fork:
https://github.com/devise-security/devise-security

**Comment by rgioia** [comment]
@natebird Nice Job!

**Comment by anuraginvest** [comment]
@natebird I would like to be added to the group too, can help with new issues. Thanks!

**Comment by anuragaryan** [comment]
@natebird I would like to be added to the group too, can help with new issues. Thanks!
